jwst/regtest/st_fitsdiff.py

```diff
@@ -1606,7 +1668,7 @@ def _report(self):
                     atol=self.atol,
                 )
 
-            if self.diff_values and self.numdiffs < self.diff_total:
+            if self.diff_values and self.numdiffs > 0 and self.numdiffs < self.diff_total:
```

Comment:
Why would `self.diff_values` be populated if `self.numdiffs == 0`, causing you to have to add a `self.numdiffs > 0` check here?